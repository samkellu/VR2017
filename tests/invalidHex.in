./binary_files/02.bin 0xs4 0xAA 0x11
