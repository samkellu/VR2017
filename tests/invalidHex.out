Delimiter byte 0 is invalid.
